service/project.go

```diff
@@ -95,6 +115,14 @@ func (uis *UIServer) projectPage(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 	}
+	grip.Debug(message.Fields{
```

Comment:
Do we want these logs to also have "func" ?

Reply:
I omitted it because these ones are just a single function